src/lib/lightning/clightning/clightningService.ts

```diff
@@ -107,11 +108,12 @@ class CLightningService implements LightningService {
     }
   }
 
-  async openChannel(
-    from: LightningNode,
-    toRpcUrl: string,
-    amount: string,
-  ): Promise<PLN.LightningNodeChannelPoint> {
+  async openChannel({
```

Comment:
Instead of adding yet another parameter to the function signature I replaced it with a single `options` parameter which contains all options which will be used to open a channel.

Reply:
Good refactor. Thanks 🥇